nuxeo-core/nuxeo-core-mongodb/src/test/java/org/nuxeo/ecm/core/mongodb/TestMongoDBIndices.java

```diff
@@ -83,11 +82,10 @@ public void shouldFailWhenCreatingExistingChildNameDocument() {
             fail("should throw a ConcurrentUpdateException");
         } catch (ConcurrentUpdateException cue) {
             assertEquals(SC_CONFLICT, cue.getStatusCode());
-            assertTrue(
-                    cue.getMessage()
-                       .contains(String.format(
-                               "E11000 duplicate key error collection: %s.test index: ecm:parentId_1_ecm:name_1 dup key: { : \"%s\", : \"%s\" }",
-                               DEFAULT_MONGODB_DBNAME, folder.getId(), DOCUMENT_NAME)));
+            String message = cue.getMessage();
+            assertTrue(message.contains("E11000 duplicate key error collection"));
+            assertTrue(message.contains(DOCUMENT_NAME));
+            assertTrue(message.contains(folder.getId()));
```

Comment:
Sorry, one last change: please put `message` as the first argument to `assertTrue`, so that if there's a mismatch we have the full message in the logs.

    assertTrue(message, message.contains("E11000 duplicate key error collection"));
    assertTrue(message, message.contains(DOCUMENT_NAME));
    assertTrue(message, message.contains(folder.getId()));